doc/installation.rst

```diff
@@ -38,75 +48,59 @@ You might need to install the ``curl`` program first, if not already installed.
 
   apt-get install curl
 
-3. Install the pre-built binary packages and its dependencies by running the following commands
+3. Install the pre-built binary packages and their dependencies by running the following commands
```

Comment:
"package" (singular) and "its" dependencies, right? Or do we still need Minuit2?

Reply:
In principle Minuit2 is not needed anymore!